src/test/server/store-test.js

```diff
@@ -0,0 +1,118 @@
+    beforeEach(() => {
+        users = usersData.map((user) => {
+            return new UserInfo(
+                {
+                    id:      user.id,
+                    name:    user.name,
+                    phone:   user.phone,
+                    email:   user.email,
+                    website: user.website
+                },
+                user.address,
+                user.company
+            );
+        });
+    });
+    describe('.getUsers()', () => {
+        it('Should return a list of users', () => {
+            assert(users, UserStore.getUsers());
+        });
+    });
+
+    describe('Actions', () => {
+        describe('addNewUser', () => {
+            it('A new user should be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Vika',
+                        phone:   '89500418181',
+                        email:   'vika.chernookaya@mail.ru',
+                        website: 'github.com'
+                    }
+                );
+                users.push(user);
+                UserAction.addNewUser(user);
+                assert(users, UserStore.getUsers());
+            });
+            it('User with incorrect fields should not be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Vika',
+                        phone:   'phone0418181',
+                        email:   'vika.chernookaya@mail.ru',
+                        website: ''
+                    }
+                );
+                UserAction.addNewUser(user);
+                assert(users, UserStore.getUsers());
+            });
+            it('Existing user should not be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Ervin Howell',
+                        phone:   '0418181',
+                        email:   'ervin@mail.ru',
+                        website: 'github.com'
+                    }
+                );
+                UserAction.addNewUser(user);
+                assert(users, UserStore.getUsers());
+            });
+            it('Existing user should not be added', () => {
+                const user = new UserInfo(
+                    {
+                        id:      11,
+                        name:    'Ervin Howell',
+                        phone:   '111111',
+                        email:   'ervin@mail.ru',
+                        website: 'github.com'
+                    }
+                );
+                users.push(user);
+                UserAction.addNewUser(user, true);
+                assert(users, UserStore.getUsers());
+            })
+        });
+        describe('updateUser', () => {
+            it('User should be updated', () => {
+                const user = new UserInfo(
```

Comment:
it's better to get initial user info, change it usign `updateUser` action and check that info changed and that the new info is not the same with initial one